### PR TITLE
Add LibreOffice

### LINKED ISSUE #46: Add libreoffice to the teamserver AMI
# 💡 Summary #

Add libreoffice to the teamserver AMI

## Motivation and context ##

Allow the operators to view word documents  that cannot be rendered in text editors/viewers

## Acceptance criteria ##

Operators would be able to open Microsoft Word documents LibreOffice

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds [LibreOffice](https://www.libreoffice.org/) to the AMI.

## 💭 Motivation and context ##

Resolves #46.

## 🧪 Testing ##

I built a new AMI for our staging Terraform environment with these changes and verified that it now contains LibreOffice.

## ✅ Pre-merge checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.
- [x] Build and test a staging AMI with these changes.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Build a new production AMI with these changes.

### CODE REVIEW

File: src/cobalt_strike.yml [hunk 1]
```diff
@@ -52,3 +52,7 @@
       vars:
         # Group ownership for the Cobalt Strike addons
         group: wheel
+    - name: Install libreoffice
```
Comment (mcdonnnj) [inline src/cobalt_strike.yml]: I think this should reference the product name and not the package.

```suggestion
    - name: Install LibreOffice
```

**Review by mcdonnnj — APPROVED** [review]
Clean and simple. One suggestion about a task description for your consideration.

File: src/cobalt_strike.yml [hunk 2]
```diff
@@ -52,3 +52,7 @@
       vars:
         # Group ownership for the Cobalt Strike addons
         group: wheel
+    - name: Install LibreOffice
+      ansible.builtin.package:
+        name:
+          - libreoffice
```
Comment (dav3r) [inline src/cobalt_strike.yml:58]: A case could be made that `cobalt_strike.yml` is no longer a great name for this playbook given the variety of tools it now installs, but that certainly doesn't need to be addressed at this moment.

**Review by dav3r — APPROVED** [review]
📚

File: src/cobalt_strike.yml (hunk 2 above)
Comment (jsf9k) [reply src/cobalt_strike.yml:58]: Feel free to create an issue to address this, uh, issue.

Comment (dav3r) [reply src/cobalt_strike.yml:58]: #48 has been issued to address this issue.

**Pull request merged**